src/Kestrel.Transport.Libuv/baseline.netcore.json

```diff
@@ -47,185 +47,6 @@
       ],
       "GenericParameters": []
     },
-    {
```

Comment:
FYI this is because of https://github.com/aspnet/KestrelHttpServer/pull/2426 which is expected.